pkg/parser/model/model.go

```diff
@@ -542,6 +542,8 @@ type TableInfo struct {
 
 	TTLInfo *TTLInfo `json:"ttl_info"`
 
+	TableVersion uint64 `json:"table_version"`
```

Comment:
Maybe `ModificationCount` is better.

Reply:
Please add a comment here. Explain the difference between "TableVersion" and "Version".